pcweb/pages/docs/components/overview.py

```diff
@@ -31,11 +33,11 @@
     definition(
         "Children",
         rx.unordered_list(
-            rx.list_item("Other Reflex components nested inside a component."),
+            rx.list_item("Text or other Reflex components nested inside a component."),
```

Comment:
The example immediately after this shows that a regular string can be passed too.